core/lexicon/en/tv_widget.inc.php

```diff
@@ -240,11 +237,25 @@
 $_lang['tv_elements_listbox-multiple_desc'] = $_lang['tv_elements_listbox_desc'];
 $_lang['tv_elements_radio_desc'] = $_lang['tv_elements_option_desc'] = $_lang['tv_elements_desc'];
 $_lang['tv_elements_tag_desc'] = $_lang['tv_elements_desc'];
-$_lang['tv_type'] = 'Input Type';
+$_lang['tv_daterange_elements_desc'] = 'Test options desc for daterange with example ph: [[+ex1]]';
+$_lang['tv_daterange_default_text_desc'] = 'Test default text desc for daterange with example ph: [[+ex1]]';
```

Comment:
These lexicons are not used in the code, they are not needed in this PR.

p.s. I ask you not to add / change lexicons without great need, problems arise:
1) New lexicons need to be translated, additional work (sometimes unnecessary), and there are not enough people. As a result, we have a semi-translated MODX, what looks crooked.
2) When a lexicon entry changes, in crowdin (https://crowdin.com/project/modx-revolution) even the translated strings are overwritten with new ones and wait for translation (because they have changed). As a result, it may turn out that after a few PRs ALL translations will be in English, because there are not enough people to translate.
3) Unused lexicons accumulate over time: extra code, extra distribution weight (see https://github.com/modxcms/revolution/issues/14512#issuecomment-650345391).

Reply:
Sorry, those were stray lex entries I was using to test something else and they never got deleted. Will remove shortly...